module/apmotel/tracer.go

```diff
@@ -86,6 +86,7 @@ func (t *tracer) Start(ctx context.Context, spanName string, opts ...trace.SpanS
 				Parent: tc,
 				Start:  startTime,
 			})
+			ctx = apm.ContextWithSpan(ctx, s.span)
```

Comment:
The problem with that is that if, someone is using both this module and the `apmot` one (it's unlikely I know), one will override the other.
I did it this way to prevent the module frome having to be exclusive with other ones using this wrapper.

Reply:
OK. TBH I don't really care about that since OpenTracing is extremely deprecated, but your approach is fine. Happy to leave it as it is. Perhaps we should drop apmot in the next major version.

Reply:
Another reason is that we have to care less about internals of the agent. We just let do its own stuff, and don't need to setup a context wrapper.